hipflowapp/Hip_Native/timer.cpp

```diff
@@ -254,10 +259,10 @@ void timinglogmsg(const char *text, uint8_t rspCount, BYTE *rspMsg)
 
 	for (i = 0; i < rspCount; i++)
 	{
-		sprintf(buf+3*i, "%02x ", rspMsg[i]);
+		sprintf_s(buf+3*i, 500, "%02x ", rspMsg[i]);
```

Comment:
500 is a hard coded number. This should should be a variable.

Reply:
replace 500 with a const int